Bottom: e6a16c9efeffaf4794c77b02ad85b3123da0cfc8
Top:    9a87e2bb1e5d76a515e2eaf6536c47201c6328df
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-03 23:51:29 +0000

Fix borast not to build for non-GL enabled builds


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 8212ceb..ac1a7f5 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -38,22 +38,6 @@ PCB_SRCS = \
 	box.h \
 	buffer.c \
 	buffer.h \
-	borast/borast-minimal.h \
-	borast/borastint-minimal.h \
-	borast/borast-bentley-ottmann.c \
-	borast/borast-combsort-private.h \
-	borast/borast-compiler-private.h \
-	borast/borast-fixed-private.h \
-	borast/borast-fixed-type-private.h \
-	borast/borast-freelist.c \
-	borast/borast-freelist-private.h \
-	borast/borast-malloc-private.h \
-	borast/borast-traps.c \
-	borast/borast-traps-private.h \
-	borast/borast-types-private.h \
-	borast/borast-wideint.c \
-	borast/borast-wideint-private.h \
-	borast/borast-wideint-type-private.h \
 	change.c \
 	change.h \
 	clip.c \
@@ -197,8 +181,23 @@ GL_SRCS= \
 	hid/common/hidgl.c \
 	hid/common/hidgl.h \
 	hid/common/trackball.c \
-	hid/common/trackball.h
-
+	hid/common/trackball.h \
+	borast/borast-minimal.h \
+	borast/borastint-minimal.h \
+	borast/borast-bentley-ottmann.c \
+	borast/borast-combsort-private.h \
+	borast/borast-compiler-private.h \
+	borast/borast-fixed-private.h \
+	borast/borast-fixed-type-private.h \
+	borast/borast-freelist.c \
+	borast/borast-freelist-private.h \
+	borast/borast-malloc-private.h \
+	borast/borast-traps.c \
+	borast/borast-traps-private.h \
+	borast/borast-types-private.h \
+	borast/borast-wideint.c \
+	borast/borast-wideint-private.h \
+	borast/borast-wideint-type-private.h
 
 BUILT_SOURCES = \
 	core_lists.h \
